### ISSUE TITLE
[Test Failure] CSharpGoToImplementation.Go To Implementation opens in a provisional tab if document wasn't already open

### ISSUE THREAD

**Issue by natidea** [dotnet/roslyn#8867]
Failure in #8832
Failing Build: http://dotnet-ci.cloudapp.net/job/dotnet_roslyn_prtest_win_vsi_p2/280/

**Comment by Pilchie** [comment]
Can you take a look?

**Comment by dpoeschl** [comment]
Related issue from the last time this failed: https://github.com/dotnet/roslyn/issues/7329

**Comment by dpoeschl** [comment]
@davkean Any idea why this build doesn't have artifacts?

**Comment by dpoeschl** [comment]
We made it always dump when this happens, and it prints "Unexpected VS notification text: a VS dump should be available" in the log... but there's no artifacts. :cry:

**Comment by davkean** [comment]
Too many builds probably. Remind me tomorrow and I'll increase the builds.

**Comment by davkean** [comment]
Yeah, there's been 50 builds since this one.

**Comment by dpoeschl** [comment]
@davkean Okay. Sadly that means I'm closing this as unactionable without the dump that we had previously added logging for.

**Comment by dpoeschl** [comment]
Another failure with artifacts (for now): http://dotnet-ci.cloudapp.net/job/dotnet_roslyn_prtest_win_vsi_p2/325/

**Comment by dpoeschl** [comment]
Artifacts saved to `<InternalShare>\public\davip\8867FailureArtifacts.zip`

**Comment by dpoeschl** [comment]
The dump is a minidump. :cry:

**Comment by dpoeschl** [comment]
I'm now pretty certain this is a dupe of #6863

**Comment by jasonmalinowski** [comment]
This is clearly not a dupe since it's still happening:

https://dotnet-ci.cloudapp.net/job/Private/job/dotnet_roslyn-internal/job/master/job/windows_vsi_p2/202/
https://dotnet-ci.cloudapp.net/job/Private/job/dotnet_roslyn-internal/job/master/job/windows_vsi_p0/178/

**Comment by natidea** [comment]
This may actually be a dupe since #6863 is itself marked as a dupe of #9602 which is still open. @jmarolf do the failures look similar? (Unfortunately, the Jenkins logs have disappeared, but I think @dpoeschl saved some logs internally.)

**Comment by natidea** [comment]
Marking this as a duplicate of #9602. Please reopen if this still occurs after #9602 is fixed.
